portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -39,18 +40,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     UserService userService = UserServiceFactory.getUserService();
 
-    final String USERNAME_FIELD = request.getParameter("username");
-    final String ID_FIELD = userService.getCurrentUser().getUserId();
+    final String usernameField = request.getParameter(USERNAME_KEY);
+    final String idField = userService.getCurrentUser().getUserId();
 
     if (!userService.isUserLoggedIn()) {
       response.sendRedirect("/login");
       return;
     }
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Entity newUser = new Entity("User", ID_FIELD);
-    newUser.setProperty("id", ID_FIELD);
-    newUser.setProperty("username", USERNAME_FIELD);
+    Entity newUser = new Entity("User", idField);
+    newUser.setProperty("id", idField);
```

Comment:
You can create an ID_KEY constant for this one so that we have consistency in that all keys have a constant representing them.

Reply:
Created id-entity-property